src/EditorFeatures/TestUtilities/Threading/WpfTestCase.cs

```diff
@@ -19,6 +20,13 @@ public class WpfTestCase : XunitTestCase
     {
         private readonly SemaphoreSlim _wpfTestSerializationGate;
 
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        [Obsolete("Called by the de-serializer; should only be called by deriving classes for de-serialization purposes")]
+        public WpfTestCase()
```

Comment:
@Pilchie, I think it needs to be public so that the serializer can properly invoke it.